examples/chart/teleport-cluster/.lint/pdb.yaml

```diff
@@ -0,0 +1,6 @@
+clusterName: helm-lint
+highAvailability:
```

Comment:
Linter is throwing another warning:

```
1 chart(s) linted, 0 chart(s) failed
--
241 | Using values from 'examples/chart/teleport-cluster/.lint/pdb.yaml': ==> Linting examples/chart/teleport-cluster
242 | [ERROR] templates/: template: teleport-cluster/templates/deployment.yaml:5:4: executing "teleport-cluster/templates/deployment.yaml" at <fail "Cannot enable multiple replicas in standalone mode, use a different chartMode which supports high availability - see README and docs">: error calling fail: Cannot enable multiple replicas in standalone mode, use a different chartMode which supports high availability - see README and docs

```

I think adding `chartMode: custom` in here will resolve this issue.

Reply:
Lets see if that does the trick.